bitnami/mariadb-galera/templates/prometheusrules.yaml

```diff
@@ -9,14 +9,10 @@ kind: PrometheusRule
 metadata:
   name: {{ template "common.names.fullname" . }}
   namespace: {{ include "common.names.namespace" . | quote }}
-  labels: {{- include "common.labels.standard" . | nindent 4 }}
-  {{- if .Values.commonLabels }}
-  {{- include "common.tplvalues.render" (dict "value" .Values.commonLabels "context" $) | nindent 4 }}
-  {{- end }}
-  {{- include "common.tplvalues.render" ( dict "value" .Values.metrics.prometheusRules.additionalLabels "context" $ ) | nindent 4 }}
-  {{- if .Values.commonAnnotations }}
-  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
-  {{- end }}
```

Comment:
I think you accidentally removed the annotations

Reply:
done at https://github.com/bitnami/charts/pull/18325/commits/71740fc1671a5e5e09921b617da0a885975576e4